simplesat/sat/policy.py

```diff
@@ -59,7 +59,11 @@ def _log_pretty_pkg_id(self, pkg_id):
         package = self._log_pool._id_to_package[pkg_id]
         name_ver = '{} {}'.format(package.name, package.version)
         fill = '.' if pkg_id % 2 else ''
-        return "{:{fill}<30} {}".format(name_ver, pkg_id, fill=fill)
+        try:
+            repo = package.repository_info.name
```

Comment:
I believe it was possible to have `repository_info` be `None` because of testing. After discussion, I'll go through and fix the tests to generate well-formed packages instead.